app/models/report/standard_form_report.rb

```diff
@@ -173,4 +173,7 @@ def can_disaggregate_with?(qing)
     qing.nil? || DISAGGABLE_TYPES.include?(qing.question.qtype_name)
   end
 
+  def references?
+    form.questionings.detect { |qing| qing.reference.present? }
```

Comment:
Note that `detect` returns a Questioning object. I think you want `any?` which returns a boolean.

Reply:
Should also use `visible_questionings` please.

Reply:
done